### ISSUE TITLE
ajax-bootstrap-select breaks required field

### ISSUE THREAD

**Issue by ppazos** [truckingsim/Ajax-Bootstrap-Select#185]
On a select with a required attribute if the "with-ajax" class is added, the required stops working and submitting the form doesn't check anything.

This plugin is breaking required fields.

Basically is causing this issue https://github.com/snapappointments/bootstrap-select/issues/514

**Comment by zkryakgul** [comment]
Did you find any solution about it?

**Comment by ppazos** [comment]
Nope. You @zkryakgul need to mention people so they get a notification. I just saw your comment from 4 years ago :)
